### PR TITLE
Attempt to partially update team listings

### PR DESCRIPTION
**Pull request by EnnuiL**
This covers the changes to the QSL team's structure, and also new folks on the QFAPI team and Mappings Triage

### CODE REVIEW

**Comment by BucketOfCompasses** [comment]
Minor nitpick but the core team is mentioned to be a subteam of QSL but isn't listed as such 

e.g. subteams have the heading "## Sub-Team: TEAM NAME HERE" where as core has the heading "## Core (Team Leaders)"

Also the overall team may need a better name as it may get confusing trying to tell if someone is referring the subteam or the overall team.

"**The QSL team** is divided into three subteams: the Core team, **the QSL team** and Quilted Fabric API."

File: src/pages/en/about/teams.mdx [hunk 1]
```diff
@@ -252,7 +252,7 @@ The Mappings team is responsible for projects relating to remapping obfuscated n
 
 The QSL team is responsible for maintaining a library of modules that provide extra APIs for modders to use when creating their mods. These APIs are designed to make certain tasks easier, and to help mods remain compatible with each other.
 
-The QSL team is divided into subteams, and each team is responsible for specific QSL modules. These modules represent specific, distinct sets of library functionality -- while they may depend on each other, they're maintained individually.
+The QSL team is divided into three subteams: the Core team, the QSL team and Quilted Fabric API.
```
Comment (Southpaw1496) [inline src/pages/en/about/teams.mdx:255]:
```suggestion
The QSL team is divided into three subteams: the Core team, the QSL team and Quilted Fabric API team.
```

**Review by Southpaw1496 — CHANGES_REQUESTED** [review]
When the team was working on the restructure, did you consider the confusion that might arise from having the QSL team as a sub team of the QSL team?

**Comment by EnnuiL** [comment]
> When the team was working on the restructure, did you consider the confusion that might arise from having the QSL team as a sub team of the QSL team?

Yes I have and I complained about the current naming scheme on an internal chat; I feel like establishing proper naming for the teams will be mandatory for this PR

**Comment by Southpaw1496** [comment]
Should it be converted to a draft, then, while you figure that out?

**Comment by EnnuiL** [comment]
> Should it be converted to a draft, then, while you figure that out?

fair enough

**Comment by Akarys42** [comment]
Closed in favor of #117

**Pull request closed without merging**